### PR TITLE
Created mobile version of the portfolio

### PR DESCRIPTION
**Pull request by ksigei**
**In this milestone, I was able to:**

- Understand how to parse a Figma design to create a UI.
- Use Flexbox to place elements on the page.
- Use CSS Grid to place elements on the page.
- Add two sections to my personal portfolio site.
      -Works section
      -About myself myself section

I also used correctly [GitHub Flow]:
I learned to write a descriptive README file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -25,11 +25,188 @@ <h1>Kiprono Japhet</h1>
...
         </div>
         
-        <button>Start collaboration</button>
+        <button type="submit">Start collaboration</button>
       </section>
         <div class="header-img-bottom">
             <img src="images/image_geometry_2.png" alt="image">
         </div>
     </header>
+  <section class="projects">
```
Comment (codepantha) [inline index.html:34]: - [ ] Please use CSS Grid to layout the main container of `<section class="projects">` according to the project requirements. 👍

File: index.html [hunk 2]
```diff
@@ -25,11 +25,188 @@ <h1>Kiprono Japhet</h1>
...
+    <div class="projects-wrapper">
+        <div class="project-image">
+            <img class="project-img" src="images/Rectangle 21 (1).png" alt="image">
```
Comment (codepantha) [inline index.html:37]: - [ ] Let's make this a background image and not an `` tag. Decorations should always background images, not `img ` tags. This [resource](https://www.w3schools.com/cssref/pr_background-image.asp) should help if you need a pointer 🤓 
- [ ] Let's align the content of this container with CSS flexbox according to the requirements ✔️

File: index.html (hunk 1 above)
Comment (codepantha) [inline index.html:34]: - [ ] You are missing a 'Projects' title for your projects section. please add a 'Projects' inside a heading tag as the title in this section.

File: index.html [hunk 3]
```diff
@@ -25,11 +25,188 @@ <h1>Kiprono Japhet</h1>
...
+        <div class="project-image">
+            <img class="project-img" src="images/Rectangle 21 (2).png" alt="image">
+        </div>
+        <div class="project-lg">
+          <div class="project-lg-1">
+            <h3>Project name goes here</h3> 
+            
+            <button type="submit" class="project-btn-1"><span class="btn-p">HTML/CSS</span></button>
+            <button type="submit" class="project-btn-1"><span class="btn-p">Ruby on Rails</span></button>
+            <button type="submit" class="project-btn-1"><span class="btn-p">JavaScript</span></button>
```
Comment (codepantha) [inline index.html:85]: - [ ] The list of project technologies (e.g., "JavaScript", "HTML" and "CSS") should be list items `<li>` tags not `<button>` inside an unordered list `ul` tag as this is the right way ✔️  to markup a list of things.

File: index.html [hunk 4]
```diff
@@ -25,11 +25,188 @@ <h1>Kiprono Japhet</h1>
...
+                
+                <span><img id="union" src="images/Union.png" alt="union"></span>
+            </button>
+          </div>
+        </div>
+    </div>
+  </section>
+  <section class="about">
+    <div class="about-me">
+        <h1>About</h1>
```
Comment (codepantha) [inline index.html:158]: - [ ] Please let's make this any other header `<hx>` tag apart from `<h1>`. You should only have one `<h1>` tag on your page and it should be around the top. Any other occurrence should be changed to between `<h2>` and `<h6>` tags 🙏

File: style.css [hunk 5]
```diff
@@ -64,25 +61,200 @@
...
+  color: #1a2236;
+}
+
+#about-view {
+  width: 288px;
+  height: 50px;
+  background: #1a2236;
+}
+
+.project-view-btn {
```
Comment (codepantha) [inline style.css:196]: - [ ] Let's make sure to implement and follow the button styling: Active, Hover and Pressed
image

File: index.html [hunk 6]
```diff
@@ -25,11 +25,188 @@ <h1>Kiprono Japhet</h1>
...
+    </div>
+    <section class="langs-section">
+        <div class="langs">
+           <div class="content-list">
+            <h3>Languages</h3>
+            <ul>
+                <li>JavaScript (ES5/ES6)</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
```
Comment (codepantha) [inline index.html:175]: - [ ] Please ensure that the technology categories ("Languages", "Frameworks" and "Skills") are `<li>` items enclosed within a `<ul>`. This is the right way to markup a list of things. 👍

File: index.html [hunk 7]
```diff
@@ -25,11 +25,188 @@ <h1>Kiprono Japhet</h1>
...
+  <section class="about">
+    <div class="about-me">
+        <h1>About</h1>
+        <P>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</P>
+
+        <button type="submit" id="about-view">
+            <span class="project-view-btn">Get my CV</span>
+            
+            <span><img id="cv-btn" src="images/ic_download.png" alt="union"></span>
+        </button>
```
Comment (codepantha) [inline index.html:165]: - [ ] Let's make sure to implement and follow the button styling: Active, Hover and Pressed
image

File: index.html (hunk 4 above)
Comment (codepantha) [inline index.html:158]: - [ ] Let's ensure that the content of this About section is aligned using CSS flexbox 🗳️

File: index.html [hunk 8]
```diff
@@ -25,11 +25,188 @@ <h1>Kiprono Japhet</h1>
...
+           <div class="content-list">
+            <h3>Languages</h3>
+            <ul>
+                <li>JavaScript (ES5/ES6)</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+            </ul>
+           </div>
+           <img class="images" src="images/#.svg" alt="#">
```
Comment (codepantha) [inline index.html]: - [ ] Please fix this image. It is currently not displaying.

File: index.html [hunk 9]
```diff
@@ -25,11 +25,188 @@ <h1>Kiprono Japhet</h1>
...
+            <ul>
+                <li>Bootstrap</li>
+                <li>Ruby on rails</li>
+                <li>RSpec</li>
+                <li>Capibara</li>
+                <li>Selenium</li>
+            </ul>
+
+            </div>
+            <img class="images" src="images/Group_18.png" alt="#">
```
Comment (codepantha) [inline index.html]: - [ ] Please fix this image. It is currently not displaying.

**Review by codepantha — CHANGES_REQUESTED** [review]
Hi @ksigei 👋 , 

You have done a great job so far. Well done 😎 
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Things to highlight ⚡ 
- Descriptive README
- Meaningful commits and PR description

## Suggested changes ♻️ 
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by ksigei** [comment]
I'm pushing now, 5 minutes 🤦

**Comment by ksigei** [comment]
Done now🤦🤦🤦🤦

**Comment by ksigei** [comment]
👋

**Comment by ksigei** [comment]
Oops 😬

**Review by ghost — CHANGES_REQUESTED** [review]
Hi @ksigei , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :stop_sign: 
### Why this happened :question: :question: 

You have submitted a project while modifying the [linters.yml](https://github.com/ksigei/Portfolio/blob/milestone2/.github/workflows/linters.yml) as well as without addressing the required changes by the previous code reviewer:

- [ ] Please get back the block of code for both `webhint` and `stylelint` into your [linters.yml](https://github.com/ksigei/Portfolio/blob/milestone2/.github/workflows/linters.yml)and make sure all checks are passed
- [ ] Please kindly address all the above-mentioned comments by the previous CR before requesting for re-review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by ksigei** [comment]
You people took time to reach out to me, the dashboard has been closed
🔒🤦💔💔💔

**Comment by ksigei** [comment]
No comment

**Comment by ksigei** [comment]
💔

**Pull request closed without merging**